api/client/network.go

```diff
@@ -113,6 +113,7 @@ func (cli *DockerCli) CmdNetworkRm(args ...string) error {
 			status = 1
 			continue
 		}
+		fmt.Fprintf(cli.out, "%s\n", net)
```

Comment:
At this place, both are OK. Using a format in `fmt.Fprintln` will make the code more flexible.